### PR TITLE
Install and configure devise

### PR DESCRIPTION
**Pull request by Ismailco**
- [x] Install devise
- [x] Setup model
- [x] Setup View

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (Mihndim2020) [inline app/models/user.rb]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md) for this project, " `Ask for confirmation of email` ", please ensure your application should ask for confirmation of email.

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```
Comment (Mihndim2020) [inline app/views/devise/sessions/new.html.erb]: - [ ] As it is [requested](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md) for this project, " `Modify the views in devise for registration and login to match the wireframes from the Sneak Peek and your styling.` ", please ensure the views in devise for the registration and login have been modified to match the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)(`checkout sneak peek: Blog app`) and their styling.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Ismailco, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Your app is running free of linter errors

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @Ismailco, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**